tox.ini

```diff
@@ -1,10 +1,10 @@
-# Copyright 2022 Canonical Ltd.
+# Copyright 2023 Canonical Ltd.
 # See LICENSE file for licensing details.
 
 [tox]
 skipsdist=True
 skip_missing_interpreters = True
-envlist = unit, integration
+envlist = unit, static, coverage-report
```

Comment:
lint is out of scope I guess

Reply:
I've added it just now. Otherwise the wf is not passing :(